configs/next/packages/mpfr/sources

```diff
@@ -20,8 +20,8 @@
 #
 
 ATSRC_PACKAGE_NAME="GNU MPFR Library"
-ATSRC_PACKAGE_VER=3.1.7
-ATSRC_PACKAGE_REV=11775
+ATSRC_PACKAGE_VER=4.1
+ATSRC_PACKAGE_REV=11945
```

Comment:
The original pull request was updating to revision 12056.   Why is this updating to an older version, now?

Reply:
Oops, not sure what happened.
Fixed now.